src/components/cards/DataSummaryCardGroup.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import CountBox from './CountBox';
+import './DataSummaryCardGroup.less';
+import { parseParamWidth } from '../../utils.js';
+
+class DataSummaryCardGroup extends Component {
+  render() {
+    const totalWidth = parseParamWidth(this.props.width);
+    return (
+      <div
+        className="data-summary-card-group"
+        style={{
+          width: totalWidth,
+          height: this.props.height,
+        }}
+      >
+        {
+          this.props.summaryItems.map((item, index) => (
+            <div
+              className={'data-summary-card-group__'.concat(this.props.connected ? 'connected' : 'separated').concat('-card')}
+              key={item.label || item[0].label}
+            >
+              {this.props.connected && index > 0
+                && <div className="left-border" />
```

Comment:
For this you just want to have all but the first child have a left border right? In your css you already have `.data-summary-card-group__connected-card:first-child { border: none; }`, so why don't you have `.data-summary-card-group__connected-card { border-left: ... }` in the css as well?

Reply:
This is because the design for border has some space between top and bottom, so only css's border attribute cannot have this effect

Reply:
Oh I just found &:first_child {border: none;} has not effects and should be deleted..Thanks for reminding me! 😂